packages/smarthr-ui/src/components/AccordionPanel/AccordionPanelContent.tsx

```diff
@@ -31,9 +31,10 @@ export const AccordionPanelContent: FC<Props & ElementProps> = ({ className, ...
   const isInclude = getIsInclude(expandedItems, name)
   const wrapperRef = useRef<HTMLDivElement>(null)
   const styles = useMemo(() => accordionPanelContent({ className }), [className])
+  const nodeRef = useRef<HTMLDivElement>(null)
 
   return (
-    <Transition in={isInclude} timeout={150}>
+    <Transition in={isInclude} timeout={150} nodeRef={nodeRef}>
```

Comment:
refs: https://reactcommunity.org/react-transition-group/transition#Transition-prop-nodeRef

> This prop is optional, but recommended in order to avoid defaulting to [ReactDOM.findDOMNode](https://reactjs.org/docs/react-dom.html#finddomnode), which is deprecated in StrictMode

とのことなので従う。 react-transition-group 自体もう2年間更新されていないので、この回避法で良さそう。